src/redux/store.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable */
+import { configureStore } from '@reduxjs/toolkit';
+import categoriesReducer from './categories/categoriesSlice';
+import bookReducer from './books/booksSlice.js';
+/* eslint-enable */
```

Comment:
- [ ] Kindly remove `eslint-disable` as it defies te purpose of using inters in the first place.

Reply:
hello sir @ShadyShawkat if i enable eslint there it asks to add file extension of import statement then when i add the .js extension for file then eslint says unexpected file extension